### DISCUSSION TITLE
How to set RWGltf_CafWriter.ToMergeFaces()  to true?

### DISCUSSION THREAD

**Discussion by qouv** [donalffons/opencascade.js#146] [Q&A] [answered]
Hello,

How can I set ToMergeFaces attribute of gltf writer to true? 

Thanks.

**Comment by donalffons** [comment, marked as answer]
Is there a problem with [`RWGltf_CafWriter::SetMergeFaces()`](https://ocjs.org/reference-docs/classes/RWGltf_CafWriter#SetMergeFaces)?

If you encounter missing functionality / bindings in OpenCascade.js, your best option is to create a [custom build](https://ocjs.org/docs/app-dev-workflow/custom-builds). Start by extending the [default build](https://github.com/donalffons/opencascade.js/blob/master/builds/opencascade.full.yml). Add classes and add methods with whatever functionality you need - bindings will be generated automatically that you can then use from JS.
